microraiden/microraiden/webui/index.html

```diff
@@ -61,13 +415,6 @@ <h5 class="text-danger">You may need to unlock your account on your Web3 provide
           </div>
         </div>
 
-        <div class="row main_switch" id="invalid_contract">
```

Comment:
This snippet shouldn't be deleted, they're required for when the user is in the wrong network (Metamask)